Mathlib/CategoryTheory/Limits/FilteredColimitCommutesFiniteLimit.lean

```diff
@@ -137,13 +137,8 @@ theorem colimitLimitToLimitColimit_injective :
     ext j
     -- Now it's just a calculation using `W` and `w`.
     simp only [Functor.comp_map, Limit.map_π_apply, curry_obj_map_app, swap_map]
-    rw [← W _ _ (fH j)]
-    rw [← W _ _ (gH j)]
-    -- Porting note: this was `simp [w]` in lean 3; this is presumably a confluence issue
```

Comment:
I didn't look into whether there is or was a confluence issue: in any case, the original proof works now.